spec/03-plugins/27-aws-lambda/99-access_spec.lua

```diff
@@ -1219,6 +1259,52 @@ for _, strategy in helpers.each_strategy() do
         assert.is_array(res.headers["Access-Control-Allow-Origin"])
       end)
     end)
+
+    describe("With latency", function()
+      lazy_setup(function()
+        assert(mock:start())
+
+        helpers.setenv("AWS_REGION", "us-east-1")
+        assert(helpers.start_kong({
+          database   = strategy,
+          plugins = "aws-lambda, http-log",
+          nginx_conf = "spec/fixtures/custom_nginx.template",
+          -- we don't actually use any stream proxy features in this test suite,
+          -- but this is needed in order to load our forward-proxy stream_mock fixture
+          stream_listen = helpers.get_proxy_ip(false) .. ":19000",
+        }, nil, nil, fixtures))
+      end)
+
+      lazy_teardown(function()
+        helpers.stop_kong()
+        helpers.unsetenv("AWS_REGION")
+        assert(mock:stop())
+      end)
+
+      it("invokes a Lambda function with GET and latency", function()
+        local res = assert(proxy_client:send {
+          method  = "GET",
+          path    = "/get",
+          headers = {
+            ["Host"] = "lambda25.test"
+          }
+        })
+
+        assert.res_status(200, res)
+        local http_log_entries
+        assert.eventually(function ()
+          http_log_entries = mock:get_all_logs()
+          return #http_log_entries >= 1
+        end).with_timeout(10).is_truthy()
+        assert.is_not_nil(http_log_entries[1])
+        local log_entry_with_latency = cjson.decode(http_log_entries[1].req.body)
+        -- Accessing the aws mock server will require some time for sure
+        -- So if latencies.kong < latencies.proxy we should assume that the
+        -- latency calculation is working. Checking a precise number will
+        -- result in flakiness here.
+        assert.True(log_entry_with_latency.latencies.kong < log_entry_with_latency.latencies.proxy)
```

Comment:
Do other places have a chance to contribute to this difference?

Reply:
Since this is a serviceless route only with lambda plugin(without upstream), the latencies.proxy should be equal to the value `KONG_WAITING_TIME` which is set inside the lambda plugin access phase